cli/standalone/serialization.rs

```diff
@@ -354,17 +356,17 @@ impl RemoteModulesStore {
 
   pub fn read<'a>(
     &'a self,
-    specifier: &'a Url,
+    original_specifier: &'a Url,
   ) -> Result<Option<DenoCompileModuleData<'a>>, AnyError> {
     let mut count = 0;
-    let mut current = specifier;
+    let mut specifier = original_specifier;
```

Comment:
We were accidentally returning the original specifier from this function instead of the resolved redirected specifeir.